test/tstCompileOnlyAccessTraits.cpp

```diff
@@ -84,4 +82,8 @@ void test_access_traits_compile_only()
   // check_valid_access_traits(PrimitivesTag{}, InvalidMemorySpace{});
 
   // check_valid_access_traits(PrimitivesTag{}, SizeMemberFunctionNotStatic{});
+
+  // check_valid_access_traits(ArborX::Traits::PrimitivesTag{}, p);
```

Comment:
Somehow, uncommenting this does not issue a deprecation warning.

Reply:
I got the warning

Reply:
Apparently, this triggers a warning for @dalg24. So, OK, still confused but should work.